code/style.css

```diff
@@ -1,19 +1,19 @@
 /* Global css variables used for colors */
 :root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --primary: #ffffff;
+  --secondary: #232440;
 }
 
```

Comment:
Excellent colors, very smooth looking fonts both aesthetically and size/weight-wise.